cli/tests/pcluster/schemas/test_cluster_schema/test_scheduler_constraints_for_custom_actions/awsbatch.on_node_configured.yaml

```diff
@@ -8,10 +8,15 @@ HeadNode:
     KeyName: ec2-key-name
   CustomActions:
     OnNodeConfigured:
-      Script: https://test.tgz  # s3:// | https://
-      Args:
-        - arg1
-        - arg2
+      Sequence:
```

Comment:
question, non-blocking: I guess you changes this yaml file but not others using custom actions right?

Reply:
not sure I get what you mean on this one

Reply:
resolved offline